Javascript(ES5)/README.md

```diff
@@ -1244,23 +1787,34 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
 
   - **Note:** IE8 and below exhibit some quirks with named function expressions.  See [http://kangax.github.io/nfe/](http://kangax.github.io/nfe/) for more info.
 
-  - If your file exports a single class, your filename should be exactly the name of the class.
-    ```javascript
-    // file contents
-    class CheckBox {
-      // ...
-    }
-    module.exports = CheckBox;
+  - Acronyms and initialisms should always be all capitalized, or all lowercased.
```

Comment:
Visitors:

- thumbs up emoji for uppercase acronyms. (`HTTPRequest`)
- thumbs down emoji for lowercase acronyms (`httpRequest`)

Comment for other suggestions. :)